storage/cloudspanner/gen.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
+cat > spanner.sdl.go <<EOF
+// Code generated by storage/cloudspanner/gen.sh DO NOT EDIT.
+
+package cloudspanner
+
+const ddlString = \`
+EOF
+cat spanner.sdl >> spanner.sdl.go
```

Comment:
This will break if anyone adds a ` char to the SDL file, a possible workaround might be something like:

```
const ddlBase64 = \"
EOF

base64 < spanner.sdl >> spanner.sdl.go

echo '\"' >> spanner.sdl.go
```

and add a test helper to unbase64 and return the the DDL to tests?

Reply:
I think this is a bit YAGNI.  The TestDDL wil lcatch this. I also haven't seen `\`` in any SDLs

Reply:
Backtick is perfectly valid DDL in the right place:

https://cloud.google.com/spanner/docs/data-definition-language#ddl_syntax
> Some words, like type names, are reserved in Cloud Spanner's DDL. If you need to use a reserved keyword as an identifier in your schema, enclose it in backticks (`).

Reply:
Fair point - done.